lib/garage/representer.rb

```diff
@@ -166,22 +166,24 @@ def encode_value(value, responder, selector)
       end
     end
 
+    PRIMITIVE_CLASSES = [
+      ActiveSupport::TimeWithZone,
+      Date,
+      Time,
+      Integer,
+      Float,
+      Hash,
+      Array,
+      String,
+      NilClass,
+      TrueClass,
+      FalseClass,
+      Symbol,
+    ]
+    PRIMITIVE_CLASSES.push(Fixnum, Bignum) unless 0.class == Integer
```

Comment:
I don't think this is necessary.
`0.is_a?(Integer)` is true in any supported Ruby versions.